backend/config/curation-api.yml

```diff
@@ -1006,6 +1000,8 @@ components:
         | `"FAILURE"` | the Dataset ran into an error during processing. See processing_status logs for more info. |
         | `"PENDING"` | the Dataset has is still being processed. |
         | `"SUCCESS"` | the Dataset has successfully been processed. |
+        | `"VALIDATION_FAILURE"` | An error occured during validation. See `processing_status_detail`|
+        | `"PIPELINE_FAILURE"`| An error occured during processing. See `processing_status_detail` |
```

Comment:
No further internal details were required to be documented for PIPELINE_FAILURE per the notes in the issue.

Reply:
```suggestion
        | `"PIPELINE_FAILURE"`| An error occured during processing. |
```